app/src/pages/api/report/list/index.ts

```diff
@@ -92,17 +36,34 @@ export default async function handler(
         return res.status(405).send({ message: METHOD_NOT_ALLOWED });
     }
 
+    /** Extract and validate queries from request sent by frontend */
+    const status = typeof req.query.status === 'string' ? req.query.status : undefined;
+    const category = typeof req.query.category === 'string' ? req.query.category : undefined;
```

Comment:
Does anything harmful happen if the API receives a bad string for status or category? Or does it just return an empty list since it's a filter?